.github/update_cache.yml

```diff
@@ -20,7 +20,7 @@ jobs:
             restore-keys: |
               ${{ runner.os }}-rmg-
         - name: Checkout RMG
-          if: steps.cache-rmg.outputs.cache-hit != 'true'
+
```

Comment:
I think these modifications should be integrated into the previous commit (or are they separated on purpose?)